docs/env.md

```diff
@@ -24,6 +24,7 @@ with a non-standard filesystem layout.
 ## daemon
 
 * `FWUPD_MACHINE_KIND` can be used to override the detected machine type, e.g. `physical`, `virtual`, or `container`
+* `FWUPD_TEST_PROFILE` can be used to load test data from `/usr/share/fwupd/test-profile.d`, e.g. `thinkpad-p1-no-iommu.json.gz`
```

Comment:
This is begging to apply into unit tests for a variety of machines!  If we store it at fwupd-test-firmware CI already clones that and can be running the battery of tests to make sure that machines security data doesn't regress different levels.

Reply:
Hilariously we have all that data already; i.e. all the HSI uploads to the LVFS.

Reply:
Ha. that's right!